pynsee/download/_load_data_from_schema.py

```diff
@@ -136,4 +136,17 @@ def _load_data_from_schema(
             usecols=variables,
         )
 
+    list_files = [file_to_import]
+    try:
+        if telechargementFichier["result"]["zip"] is True:
+            list_files += [dataPathFile,
+                           f"{zipDirectory}/" + telechargementFichier["file_archive"],
+                           telechargementFichier["file_archive"]]
+        
+        for f in list_files:
+            if os.path.exists(f):
+                os.remove(f)
+    except:
```

Comment:
Better not to have bare excepts: if I'm not mistaken, the only potential error here is from ``os.remove``, right?
In that case it would be better to wrap that one line, so that if one file fails, later files are still removed.
The error raised should be ``FileNotFoundError`` if the file does not exist, there may be other if the user does not have write access.

Reply:
indeed, I was mostly worried about the access rights

Reply:
this commit d201b0795428266de9ac0b95e4f7b6179400f2a7 addresses the issue